server/services/communityApiService.ts

```diff
@@ -78,4 +79,28 @@ export default class CommunityApiService {
       throw createError(err.status, err, { userMessage: 'Could retrieve staff details from nDelius.' })
     }
   }
+
+  async getResponsibleOfficersForServiceUser(crn: string): Promise<DeliusOffenderManager[]> {
+    const token = await this.hmppsAuthService.getApiClientToken()
+
+    logger.info({ crn }, 'getting offender managers for service user')
+    try {
+      const deliusOffenderManagers = (await this.restClient.get({
+        path: `/secure/offenders/crn/${crn}/allOffenderManagers`,
+        token,
+      })) as DeliusOffenderManager[]
+
+      const responsibleOfficer = deliusOffenderManagers.filter(offenderManager => offenderManager.isResponsibleOfficer)
+
+      if (!responsibleOfficer) {
+        throw createError(500, `No offender manager found in Delius for CRN ${crn}`, {
+          userMessage: 'No responsible officer is assigned to this service user in nDelius.',
+        })
+      }
+
+      return responsibleOfficer
+    } catch (err) {
+      throw createError(err.status, err, { userMessage: 'Could retrieve offender managers from nDelius.' })
```

Comment:
is 'offender manager' language SPs are familiar with?

Reply:
I've updated this to say 'Could not retrieve Responsible Officer from nDelius' instead.